data/tweets/geo_covid_tweet_ids/geo/sc.txt

```diff
@@ -1 +1,2 @@
 1260291666130489346
+1260291666130489346
```

Comment:
Was this added by accident?

Reply:
Yeah, not sure how it happened. Will delete the file.